### DISCUSSION TITLE
Automation Send F5 with VNC Viewer

### DISCUSSION THREAD

**Discussion by JeedHome44** [Poeschl/Hassio-Addons#363] [Q&A]
Hello,

With your addon, can we do automation to connect in a Magic Mirror and do F5 and next disconnect ?
I need to do F5 in 2 Magic Mirror to refresh it when i want.

Thank you.

PS: Sorry for my bad english, i'm french

**Comment by Poeschl** [comment]
The addon only allows to interact with an vnc server manually. An automation with it is not possible (my current state of knowledge), sry.

**Comment by JeedHome44** [comment]
is there an addon allowing it ?

**Comment by Poeschl** [comment]
@JeedHome44 Not that I know of. I also don't know a VNC client which can transmit keystrokes.
Maybe you have other options like http requests or mqtt commands to refresh the screen.
